packages/runtime/src/lib/constants.ts

```diff
@@ -8,36 +8,51 @@ export const MAX_CONCAT_SIZE = 4096;
 export const SSC_BYTES = 1; // 1 represents type 'byte' in stateful key-value pair
 export const SSC_UINT = 2; // 2 represents type 'uint' in stateful key-value pair
 export const ALGORAND_MIN_TX_FEE = 1000;
+
+// values taken from https://github.com/algorand/go-algorand/find/master#L689
+export const LogicSigMaxCost = 20000;
+export const MaxAppProgramCost = 700;
+export const LogicSigMaxSize = 1000;
+export const MaxAppProgramLen = 1024;
```

Comment:
just to confirm. we don't have this for `MaxClearProgramLen` right ? same for cost

Reply:
Yes, i couldn't find specfic max lengths for approval and clear programs in go-algorand. Please let me know if you will find it.

Reply:
i think it's same for both.

Reply:
i think it's same for both.